docs/themes/code_mode.rst

```diff
@@ -1,6 +1,52 @@
 .. vale off
 
-Code Mode
+Code mode
 ##########
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+Code Mode is an option available in the Email and Landing Page edit Form. It allows you to create/insert/edit your content in HTML code. It's helpful for situations where you don't want to use a Mautic Theme and you want to use an HTML Theme copied from a ``3rd`` party Theme builder or if you enjoy editing HTML code so much.
+
+With Mautic 2.3.0, Code Mode replaced the full-page Froala (WYSIWYG) editor that sometimes altered the HTML code. Code Mode won't modify any of the code you paste in. The other option to edit a page/email content is to use the Builder. It uses Froala editor only to edit the text/image content, not the full page, so that it won't modify the page layout.
```

Comment:
```suggestion
Code mode won't modify any of the code you paste in. The other option to edit a page/email content is to use the Builder. It uses CKEditor only to edit the text/image content, not the full page, so that it won't modify the page layout.
```

I don't think that it is relevant nowadays to have reference to such an old version - especially as we don't use Froala any more (we use CKEditor now).